crates/apub/src/activity_lists.rs

```diff
@@ -117,7 +117,8 @@ impl InCommunity for AnnouncableActivities {
       CollectionRemove(a) => a.community(context).await,
       LockPost(a) => a.community(context).await,
       UndoLockPost(a) => a.community(context).await,
-      Page(_) => unimplemented!(),
+      // TODO this needs to be implemented
```

Comment:
Doesnt need to be implemented. We only send out `Annunce/Page` activities for compat with Mastodon, but ignore it for incoming activities because we also send them as `Announce/Create/Page`.